apps/service_define/lib/define_web/endpoint.ex

```diff
@@ -0,0 +1,37 @@
+defmodule DefineWeb.Endpoint do
+  use Phoenix.Endpoint, otp_app: :service_define
+
+  @session_options [
+    store: :cookie,
+    key: "_define_key",
+    signing_salt: "+YSBmf7s"
+  ]
+
+  socket "/socket", DefineWeb.UserSocket,
+    websocket: true,
+    longpoll: false
+
+  plug Plug.Static,
+    at: "/",
+    from: :service_define,
+    gzip: true
+
+  if code_reloading? do
+    socket "/phoenix/live_reload/socket", Phoenix.LiveReloader.Socket
+    plug Phoenix.LiveReloader
+    plug Phoenix.CodeReloader
+  end
```

Comment:
do we want to leave this in long-term since it's essentially only for testing purposes?